libraroo-web/src/bookCatalog/components/CreateBookCatalog.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import BookCatalogService from "../services/bookCatalog.service";
```

Comment:
Change to bookCatalogService as this is a instance of a class

Reply:
yes okay yes